### DISCUSSION TITLE
Can I load all tokens and NFTs from the connected wallet?

### DISCUSSION THREAD

**Discussion by JJ810** [wevm/wagmi#216] [Q&A]
In my application we are showing all tokens and NFTs from connected wallet and we were using some other library to do this. But after playing around with `wagmi`, I liked it a lot and wanted to switch to `wagmi`. And so I wanted to ask if I can get all tokens and NFTs from the connected wallet?

**Comment by humblecoder** [comment]
Yup, I need the same thing.  Any word on this?

What is the current recommended mechanism for retrieving/listing all token balances of a given wallet?  As it stands `useBalance()` `data` seems to only return data for the native (or specified) token.  I need all balances.

I realize different tools work differently, but basically I'm looking for the equivalent of [useERC20Balances](https://github.com/MoralisWeb3/react-moralis#useerc20balances) in `react-moralis`

**Comment by wr1159** [comment]
bump, is there a solution for this problem now?
